src/app/store/trips/effects.ts

```diff
@@ -52,7 +52,8 @@ export class TripsEffects {
           .addTrip(action.trip)
           .then(data => {
             if (!data.id) {
-              return createNewTripComplete({ trip: {} as Trip });
+              alert(`Error creating new trip. No document ID returned.`);
```

Comment:
What does a user know about document ids? Make this readable by your girlfriend 🤣